src/App.js

```diff
@@ -340,6 +342,19 @@ class App extends React.Component {
     return this.state.wrapper.requestAddressIdentityModification(address)
   }
 
+  renderBanner(mode = 'wrapper') {
+    const { showDeprecatedBanner, showUpgradeBanner, locator } = this.state
+    if (showDeprecatedBanner) {
+      return (
+        <DeprecatedBanner dao={locator.dao} lightMode={mode === 'onboarding'} />
```

Comment:
Maybe we can just remove this component entirely 😄

Reply:
Yes! :tada: